<launch>

    <node pkg="usv_simulation" type="los.py" name="los" />

	<node pkg="usv_simulation" type="boat_model" name="boat_model" />

	<node pkg="usv_simulation" type="boat_tf2_broadcaster" name="boat_tf2_broadcaster" />

    <node name="asmc_simulation" pkg="usv_simulation" type="asmc_simulation" >
	<param name = "k_u" value = "0.1" />
	<param name = "k_psi" value = "0.2" />
	<param name = "kmin_u" value = "0.05" />
	<param name = "kmin_psi" value = "0.2" />
	<param name = "k2_u" value = "0.02" />
	<param name = "k2_psi" value = "0.1" />
	<param name = "mu_u" value = "0.05" />
	<param name = "mu_psi" value = "0.1" />
	<param name = "lambda_u" value = "0.001" />
	<param name = "lambda_psi" value = "1" />
    </node>

</launch>